### DISCUSSION TITLE
Prefab/Template Request

### DISCUSSION THREAD

**Discussion by jackyyang09** [GitbookIO/community#401] [Feature Requests]
I'd like to use Gitbook as a way to host content about my video game project in the style of a "wiki" with the help of Template or "Prefab" pages.

As an example, my game features characters, each character page will then have a Name field, an Image, and a short Description.

Later in development, I might decide to add a new field to my character pages, but if I already have pages created, I would need to go back to each page and copy+paste the new field to each of them. 

Please add Prefabs/Templates and allow changes to said Prefabs/Templates to be reflected in whatever page inherits from them.

**Comment by KennyTech** [comment]
I read something about this or a variant of it on the roadmap last year. 
I'm not sure why it was on the roadmap for so long and there's no sign of it now?
Hope this quality of life feature goes back on the roadmap at a higher priority.

**Comment by aikoii** [comment]
Hey @jackyyang09 and @KennyTech, thanks for sharing your thoughts! I encourage you to join similar discussions or vote 👉 #121 and #374. I'll be closing this in favor of those. 

Regarding the roadmap inclusion of this feature, I acknowledge that it was previously mentioned in our development plans. However, due to focusing on other priorities, it's not on our immediate roadmap 🙏. But it remains a part of our future plans.

**Comment by mcgaritydotme** [comment]
@jackyyang09 By "template", do you mean just a starting format so you don't have to create a page from a blank slate?  Or do you mean being able to define content once and reuse it on pages?  It sounded like the former, which is different than #121 that covers the latter.

**Reply by jackyyang09** [reply]
I'm referring to a dynamic "template" that isn't only applied once, but rather a page with a dynamic layout, whose contents are moved around as the "template" itself is changed. Where issue #121 is a feature request for shared content, I'm looking for a shared page layout.
